ci/set-terraform-version

```diff
@@ -1,2 +0,0 @@
-#!/bin/sh
-ln -fs ~/.local/bin/terraform12 ~/.local/bin/terraform
```

Comment:
Why is this no longer needed? Would be nice to know the rationale.
Also, the CI code seems to still reference this script.

Reply:
Actually looks like CI is failing because of this change.

Reply:
As mentioned in the comment, CI changes needs to be applied first, so this PR passes the checks. I'll add some comment about removal here too, as it's done in CI code.

Reply:
Done.